tests/transforms/test_mitigate.py

```diff
@@ -127,7 +132,7 @@ def test_multi_returns(self):
             qml.transforms.mitigate_with_zne,
             scale_factors=[1, 2, 3],
             folding=fold_global,
-            extrapolate=richardson_extrapolate,
+            extrapolate=exponential_extrapolate,
```

Comment:
In this test file can you parametrize the extrapolation then we cover both?

Reply:
Good call. Done in https://github.com/PennyLaneAI/pennylane/pull/5972/commits/31aa0e4bd85d4948963e4311905aa9755734682d.